docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -152,39 +153,45 @@ To illustrate some of these improvements, the following table display a benchmar
+We will also keep expanding our portfolio of charts, with types such as [Heat map](/x/react-charts/heat-map/), [Funnel](/x/react-charts/funnel/), and [Gantt](/x/react-charts/gantt/), and explore virtualization and other advanced use cases for the Tree view component.
 
-We encourage you to upvote issues on GitHub to help us prioritize. Your input directly influences our development schedule, so don't hesitate to let us know what matters most to you!
-You can use the following list for quick access to each components issues.
+We encourage you to upvote issues on GitHub to help us prioritize.
+Your input directly influences our development schedule, so don't hesitate to let us know what matters most to you!
+Use the following list to quickly access each component's issues:
 
 - [Charts](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+charts%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 - [Tree View](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+treeview%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 - [Data Grid](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+data+grid%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 - [Date and Time pickers](https://github.com/mui/mui-x/issues?q=is%3Aissue+is%3Aopen+label%3A%22component%3A+pickers%22+label%3A%22waiting+for+%F0%9F%91%8D%22)
 
-The next major release (v7) will be available through alpha and beta versions until March 2024, when it's planned for it to be officially promoted to the new stable version. At the same time, v6 will transition to [Long-Term Support](/versions/#long-term-support-lts), while support for v5 will be discontinued.
+The next MUI X major version, v7, is available through alpha and beta versions until March 2024, when we plan to promote it to stable.
+At the same time, v6 will transition to [Long-Term Support](/versions/#long-term-support-lts), while support for v5 will be discontinued.
 
-## Feedback
+## Give us feedback
 
-As usual, we are happy to hear from you and get your feedback, so if you'd like to share your pain points and use cases, please leave your contact info through [this Google Form](https://forms.gle/vsBv6CLPz9h57xg8A).
+We're excited to hear from you about improving MUI X!
+Please share your requests, suggestions, pain points, and use cases through [this Google Form](https://forms.gle/vsBv6CLPz9h57xg8A).
 
-Naturally, you're welcome to request new features, report bugs, and join any discussion in MUI X [GitHub repository](https://github.com/mui/mui-x/issues); additionally, we're trying to establish a community in the [MUI Discord server](https://mui.com/r/discord/), and we'd love to see you there.
+Or, as always, you're welcome to request new features, share bug reports, and join any discussions in [the MUI X GitHub repository](https://github.com/mui/mui-x/issues).
```

Comment:
I think "we're trying to establish a community" felt weird and low-key 😅 
We have a community, and a lot a people are there already! We're just trying to establish a cadence of actually using Discord... but that can be omitted 😬